src/clj/rems/db/fix_userid.clj

```diff
@@ -0,0 +1,346 @@
+
```

Comment:
tests would be good to write for this functionality at some point

Reply:
I was thinking that we write the tests while developing the more comprehensive userid rename support i.e. https://github.com/CSCfi/rems/blob/master/docs/architecture/014-userids.md

For some of the "missing functionality" like update functions we could do it now however they are also not used anywhere else yet.

Reply:
So manual testing so far. Script also records everything it did.